webapp/src/features/adminOrganizationTab/AdminOrganizationTab.js

```diff
@@ -0,0 +1,193 @@
+import React, { useEffect } from "react";
+import AdminAddOrganization from "../adminAddOrganization/AdminAddOrganization";
+import AdminOrganizationTabRsu from "../adminOrganizationTabRsu/AdminOrganizationTabRsu";
+import AdminOrganizationTabUser from "../adminOrganizationTabUser/AdminOrganizationTabUser";
+import AdminEditOrganization from "../adminEditOrganization/AdminEditOrganization";
+import AdminOrganizationDeleteMenu from "../../components/AdminOrganizationDeleteMenu";
+import { IoChevronBackCircleOutline, IoRefresh } from "react-icons/io5";
+import { AiOutlinePlusCircle } from "react-icons/ai";
+import Grid from "@mui/material/Grid";
+import EditIcon from "@mui/icons-material/Edit";
+import { DropdownList } from "react-widgets";
+import {
+  selectActiveDiv,
+  selectTitle,
+  selectOrgData,
+  selectSelectedOrg,
+  selectRsuTableData,
+  selectUserTableData,
+  selectErrorState,
+  selectErrorMsg,
+
+  // actions
+  editOrg,
+  deleteOrg,
+  getOrgData,
+  updateTitle,
+  setActiveDiv,
+  setSelectedOrg,
+} from "./adminOrganizationTabSlice";
+import { useSelector, useDispatch } from "react-redux";
+
+import "../adminRsuTab/Admin.css";
+
+const AdminOrganizationTab = (props) => {
+  const dispatch = useDispatch();
+  const activeDiv = useSelector(selectActiveDiv);
+  const title = useSelector(selectTitle);
+  const orgData = useSelector(selectOrgData);
+  const selectedOrg = useSelector(selectSelectedOrg);
+  const rsuTableData = useSelector(selectRsuTableData);
+  const userTableData = useSelector(selectUserTableData);
+  const errorState = useSelector(selectErrorState);
+  const errorMsg = useSelector(selectErrorMsg);
+
+  let orgPatchJson = {
+    orig_name: selectedOrg.name,
+    name: selectedOrg.name,
+    users_to_add: [],
+    users_to_modify: [],
+    users_to_remove: [],
+    rsus_to_add: [],
+    rsus_to_remove: [],
+  };
+
+  const updateOrgData = async (specifiedOrg) => {
+    dispatch(getOrgData({ orgName: "all", all: true, specifiedOrg }));
+  };
+
+  useEffect(() => {
+    updateOrgData();
+  }, []);
```

Comment:
Include updateOrgData in dependency array?

Reply:
added